### PR TITLE
Add process info

### LINKED ISSUE #58: Report process info
gops can report more process info (e.g. https://godoc.org/github.com/shirou/gopsutil/process) for a PID. The only downside is that this command will be asymmetrical because we cannot provide the same functionality for the processes working remotely. 

```
$ gops <pid>
Create time: XXX
Number of threads: 4
Number of context switches: X
Number of file descriptors: X
Number of open files: X

Connections
...
```

@odeke-em, what do you think?

### PR DESCRIPTION
**Pull request by rakyll**
Example output:

$ gops <pid>
threads:      14
memory usage: 0.090%
cpu usage:    0.088%
username:     jbd
cmd+args:     /Users/jbd/bin/gocode -s -sock unix -addr 127.0.0.1:37373

Fixes #58.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -8,16 +8,20 @@ package main
...
 	"strconv"
 	"strings"
 
 	"github.com/google/gops/goprocess"
+	"github.com/shirou/gopsutil/process"
 )
 
 const helpText = `Usage: gops is a tool to list and diagnose Go processes.
 
+	go [cmd] ...
```
Comment (odeke-em) [inline main.go]: s/go /gops /

File: main.go [hunk 2]
```diff
@@ -8,16 +8,20 @@ package main
...
+	go <pid> # displays process info
```
Comment (odeke-em) [inline main.go]: ditto about s/go /gops /

File: main.go [hunk 3]
```diff
@@ -46,12 +50,17 @@ func main() {
 	}
 
 	cmd := os.Args[1]
+
+	// see it is a PID
```
Comment (odeke-em) [inline main.go]: nit: see if it is a PID

File: main.go (hunk 1 above)
Comment (rakyll) [reply main.go]: Done

File: main.go (hunk 2 above)
Comment (rakyll) [reply main.go]: Done

File: main.go (hunk 3 above)
Comment (rakyll) [reply main.go]: Done.

**Comment by rakyll** [comment]
PTAL, all addressed.

**Pull request merged**